packages/editor-ui/src/components/MainSidebar.vue

```diff
@@ -323,9 +328,21 @@ onClickOutside(createBtn as Ref<VueInstance>, () => {
 				@select="handleMenuSelect"
 			>
 				<N8nIconButton icon="plus" type="secondary" outline />
+				<template #[`item.append.${CREATE_PROJECT_ID}`]="{ item }">
```

Comment:
Just spotted this
Isn't there a nicer solution?
This looks very weird, could it be just a computed?

Reply:
it's a dynamic slot and is used for thing like this when you only want your changes to affect specific elements within what's rendered,

i could create a computed inside the composable that return the same string, but it would be just moving it from one place to another

Reply:
Sure, but I think it's more readable and understandable

Reply:
you were right it looks much nicer, thank you!